discordClient.js

```diff
@@ -27,7 +27,7 @@ discordClient.on("ready", function () {
     var guilds = discordClient.guilds.array();
     guilds.forEach(function (guild) {
         var val = Math.floor(Math.random() * randomFunnyMessages.length);
-        guild.defaultChannel.send(randomFunnyMessages[val]);
+        guild.channels.find("id", consts.ChannelId).send(randomFunnyMessages[val]);
```

Comment:
out of interest, any way to find by name rather than by id?
might make it easier to know what it's posting to.